.github/workflows/publish-quil-py.yml

```diff
@@ -83,10 +93,11 @@ jobs:
 
   windows: 
     runs-on: windows-latest 
+    needs: [ is-python-release, should-publish-wheels ]
     strategy: 
       matrix: 
         python-version: ['3.8', '3.9', '3.10', '3.11'] 
```

Comment:
Should we add `3.12` now that it's stable?

Reply:
Here are those issues:
https://github.com/rigetti/rigetti-pyo3/issues/32
https://github.com/rigetti/quil-rs/issues/305
https://github.com/rigetti/qcs-sdk-rust/issues/382